doc/changelog/11-infrastructure-and-dependencies/12583-fix-remake.rst

```diff
@@ -0,0 +1,5 @@
+- **Fixed:**
+  Running ``make`` in ``test-suite/`` twice (or more) in a row will no longer
+  rebuild the ``modules/`` tests on subsequent runs, if they have not been
+  modified in the meantime (`#12583 <https://github.com/coq/coq/pull/12583>`_,
+  fixes `#12582 <https://github.com/coq/coq/issues/12582>`_, by Jason Gross).
```

Comment:
Is this really worth a changelog?

Reply:
I can delete it if you'd prefer?  I've been erring on the side of more changelog rather than less, but I'm fine either way.